[terra-show-hide] prop to focus on the element added (#3838)
cerner/terra-core: packages/terra-show-hide/CHANGELOG.md

## Unreleased

* Added

>   * New focusRef prop to shift focus to upon revealing the hidden content for screen-reader users.

Reviewer: Can we reword to:
```suggestion
  * Added `focusRef` prop to shift focus to the hidden content when it is revealed for screen-reader users.
```

Author: Good catch! Thank you for suggesting. Added here: https://github.com/cerner/terra-core/pull/3838/commits/848c29f87e0af5b43f6eaade4494fd8a82d0ba6e
